### PR TITLE
[DD-446] Implement option to disable read time

### PR DESCRIPTION
**Pull request by chantelle11**
Ticket [Link](https://circleci.atlassian.net/browse/DD-446)

# Description
In [DD-425: Move the "top reasons why my build failed..." from support to docDEV IN PROGRESS](https://circleci.atlassian.net/browse/DD-425) we want to disable the read-time because it might not completely make sense for this page (no one read the entire page). We should add an option as a front matter variable to let this be customizable.

# Changes
- A variable called readTime: false can be set in a markdown page to disable the read time for the output page
- by defaults readTime needs to be true for all pages (in _config).

# How to test
You will have to test it locally by adding in new `readTime` value into md file
Screen Shot 2022-03-02 at 3 37 06 PM

# Screenshots
## Before
Screen Shot 2022-03-02 at 3 42 30 PM

## After
Screen Shot 2022-03-02 at 3 52 47 PM

### CODE REVIEW

File: jekyll/_includes/quick-info-list.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <div id='arrow' data-popper-arrow></div>
 </div>
 <div class="quick-info-list">
+  {% if page.readTime != false %}
   <img alt="Last updated" src="{{ site.baseurl }}/assets/img/docs/duration.svg">
   <time aria-describedby="tooltip-time" id="time-posted-on" datetime="{{  page.last_modified_at | date_to_xmlschema }}">
```
Comment (rsylvian) [inline jekyll/_includes/quick-info-list.html:10]: The readtime option should not hide the Last updated date

**Review by rsylvian — CHANGES_REQUESTED** [review]

File: jekyll/_includes/quick-info-list.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
   <div id='arrow' data-popper-arrow></div>
 </div>
 <div class="quick-info-list">
+  {% if page.readTime != false %}
```
Comment (teesloane) [inline jekyll/_includes/quick-info-list.html:8]: Super nit, but I feel that `!= false` is a bit tricky to read. Is it possible to just use `{% if page.readTime %}` in liquid? And if not ... just use `{% if page.readTime == true %}`?

Comment (rsylvian) [reply jekyll/_includes/quick-info-list.html:8]: Yup great point!

Comment (chantelle11) [reply jekyll/_includes/quick-info-list.html:8]: I was following how we do page.toc I can update no worries! I agree makes sense

**Review by rsylvian — APPROVED** [review]
Looks good ✅

**Review by tshen815 — APPROVED** [review]
nice work!

**Pull request merged**